hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/DeploymentManagementTest.java

```diff
@@ -561,7 +561,7 @@ public void assignDistributionSetAndAutoCloseActiveActions() {
             @Expect(type = TargetUpdatedEvent.class, count = 20), @Expect(type = ActionCreatedEvent.class, count = 20),
             @Expect(type = DistributionSetCreatedEvent.class, count = 2),
             @Expect(type = SoftwareModuleCreatedEvent.class, count = 6),
-            @Expect(type = MultiActionEvent.class, count = 2),
+            @Expect(type = MultiActionAssignEvent.class, count = 2),
```

Comment:
If you distinguish between assignment and cancelation now, this should be reflected in tests as well. So e.g. we would expect to see no `MultiActionCancelEvent` here

Reply:
... furthermore I miss a test where now actually a `MultiActionCancelEvent` is expected